src/Domain/Structure/OrientationMap.cpp

```diff
@@ -92,25 +94,91 @@ template <size_t VolumeDim>
 OrientationMap<VolumeDim> OrientationMap<VolumeDim>::inverse_map() const {
   std::array<Direction<VolumeDim>, VolumeDim> result;
   for (size_t i = 0; i < VolumeDim; i++) {
-    gsl::at(result, gsl::at(mapped_directions_, i).dimension()) =
-        Direction<VolumeDim>(i, gsl::at(mapped_directions_, i).side());
+    gsl::at(result, get_direction(i).dimension()) =
+        Direction<VolumeDim>(i, get_direction(i).side());
   }
   return OrientationMap<VolumeDim>{result};
 }
 
 template <size_t VolumeDim>
 void OrientationMap<VolumeDim>::pup(PUP::er& p) {
-  size_t version = 0;
-  p | version;
   // Remember to increment the version number when making changes to this
   // function. Retain support for unpacking data written by previous versions
   // whenever possible. See `Domain` docs for details.
-  if (version >= 0) {
-    p | mapped_directions_;
-    p | is_aligned_;
+  const uint16_t version = 1;
+  if (p.isUnpacking()) {
```

Comment:
This feels like it needs a test to ensure old versions are deserialized properly.

Reply:
I disagree, we just need to stop versioning tiny classes, anything else is a bandaid and not scalable. We can't version every single 1 byte class, in part because it adds huge code maintenance burdens. See https://github.com/sxs-collaboration/spectre/issues/5986